samples/apps/copilot-chat-app/webapi/Controllers/ChatParticipantController.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SemanticKernel.Service.Model;
+using SemanticKernel.Service.Storage;
+
+namespace SemanticKernel.Service.Controllers;
+
+/// <summary>
+/// Controller for chat invitations.
+/// This controller is responsible for:
+/// 1. Creating invitation links.
+/// 2. Accepting/rejecting invitation links.
+/// 3. Managing participants in a chat session.
+/// </summary>
+[ApiController]
+[Authorize]
+public class ChatParticipantController : ControllerBase
```

Comment:
It used to be `ChatInvitation`. We decided to use `ChatParticipant` instead since it's not really an invitation system. It doesn't have an invitation object that it accepts when the APIs is invoked. Instead, it creates a new participant.